core/domain/action_registry.py

```diff
@@ -35,7 +35,7 @@ def get_all_action_ids(cls):
         Returns:
             list(str). The list of all allowed action IDs.
         """
-        return feconf.ALLOWED_ACTION_IDS
+        return feconf.ALLOWED_ACTION_TYPES
```

Comment:
docstring says "action IDs", this says action types, please standardize.

Reply:
Done.